src/errors.rs

```diff
@@ -25,7 +25,7 @@ pub enum Error {
     /// An error containing a [`protobuf::error::ProtobufError`].
     #[cfg(feature = "protobuf")]
     #[error("Protobuf error: {0}")]
-    Protobuf(#[from] protobuf::error::ProtobufError),
+    Protobuf(#[from] protobuf::Error),
```

Comment:
The docstring above needs to be updated in the same way too.

Reply:
fixed with aae3a0df414fc6a7fe36a54c773e7d0a0c66ad12